### ISSUE TITLE
PostCSS version update

### ISSUE THREAD

**Issue by paulkre** [swashata/wp-webpack-script#1183]
Is it possible to update the version of `postcss-loader` in `@wpackio/scripts`? It's currently at v3 but the latest version is already v5.

Love the project btw! 💚

**Comment by swashata** [comment]
Thank you for the kind words. I will very soon do a feature update. But in the meanwhile, you can send me a PR too. If everything works, the e2e tests should pass.

**Comment by peterwhite** [comment]
Is there any update on this? With TailwindCSS requiring PostCSS8 in future versions, I'd love to use it with wpackio. 

Thanks in advance - very excited about the future of this project!

**Comment by drivenfaroff** [comment]
Running into same issue here where something in the @wpackio/scripts doesn't seem to like PostCSS 8 when I install it for TailwindCSS.

I had to install the [TailwindCSS PostCSS 7 Compatibility Build](https://tailwindcss.com/docs/installation#post-css-7-compatibility-build) for now.

**Comment by paulkre** [comment]
I modified the webpack config in order to make TailwindCSS work. It's kind of ugly though.

First I installed the correct dependency versions for Tailwind: `yarn add postcss@^8 postcss-loader@^4`

Then I modified the `wpackio.project.js`:

```js
module.exports = {
  // ...
  files: [
    {
      // ...
      webpackConfig: (config) => ({
        ...config,
        module: {
          ...config.module,
          rules: config.module.rules.map((rule) =>
            rule.test.toString() === "/\\.css$/"
              ? {
                  ...rule,
                  use: rule.use.map((loaderConfig) =>
                    loaderConfig.loader.includes("postcss-loader")
                      ? {
                          ...loaderConfig,
                          loader: require.resolve("postcss-loader"),
                        }
                      : loaderConfig
                  ),
                }
              : rule
          ),
        },
      }),
    },
  ],
};
}
```

**Comment by swashata** [comment]
Hello,

I have released version 6.0 which updates `postcss-loader`. Now `postcss` is a peer dep. So once you update to v6, do a `yarn bootstrap` and it will install the latest needed dependencies (including postcss 8).
